.github/workflows/ci.yml

```diff
@@ -205,7 +216,6 @@ jobs:
       fail-fast: false
       matrix:
         modules:
-          - ":presto-docs"
```

Comment:
`:presto-docs` test didn't fail:   https://github.com/prestodb/presto/pull/15755/checks?check_run_id=1982707040 

Is this PR an improvement rather than fixing something broken?

Reply:
It's not that the docs fail it's that when they fail the job still is marked as a success. It also addresses the issue where it retries rebuilding the docs for 30 minutes. 

However I am not actually sure the first thing I said is true, and the second thing goes away in a different PR where I remove all the retries. I think I will close this for now.